go/fn/runnerProcessor.go

```diff
@@ -39,9 +40,19 @@ func (r *runnerProcessor) config(ctx *Context, o *KubeObject) {
 		data := o.NestedStringMapOrDie("data")
 		fnRunnerElem := reflect.ValueOf(r.fnRunner).Elem()
 		for i := 0; i < fnRunnerElem.NumField(); i++ {
-			if fnRunnerElem.Field(i).Kind() == reflect.Map {
-				fnRunnerElem.Field(i).Set(reflect.ValueOf(data))
-				break
+			switch fnRunnerElem.Field(i).Kind() {
+			case reflect.String:
+				for k, v := range data {
+					lowerKey := strings.ToLower(k)
```

Comment:
I like this approach.  We might consider also using the `json` tag to get the ConfigMap field name.